system/modules/isotope/library/Isotope/CompatibilityHelper.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+/*
+ * Isotope eCommerce for Contao Open Source CMS
+ *
+ * Copyright (C) 2009 - 2019 terminal42 gmbh & Isotope eCommerce Workgroup
+ *
+ * @link       https://isotopeecommerce.org
+ * @license    https://opensource.org/licenses/lgpl-3.0.html
+ */
+
+namespace Isotope;
+
+use Contao\System;
+use Symfony\Component\HttpFoundation\Request;
+
+/**
+ * Class CompatibilityHelper
+ *
+ * Provides compatibility-related methods for Contao Isotope
+ *
+ */
+class CompatibilityHelper
+{
+    /**
+     * Get the current request.
+     *
+     * @return Request|null
+     */
+    public static function getCurrentRequest()
```

Comment:
if the method is not used publicly, it should be private only for this class. We try not to expose unecessary API because people might use it and we'll never be able to change it afterwards.

Reply:
Ok, that's a good reason to keep the method private. If it shows up being useful later, we can change it to public.

Reply:
exactly